person_class.rb

```diff
@@ -1,6 +1,6 @@
 class Person
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- [ ] After creating getters and setters method for rentals, it then needs to be initialized. You can read more about it [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)